ocrd_tesserocr/binarize.py

```diff
@@ -72,16 +71,15 @@ def process(self):
                                                                          value=self.parameter[name])
                                                                for name in self.parameter.keys()])]))
                 page = pcgts.get_Page()
-                page_image = self.workspace.resolve_image_as_pil(page.imageFilename)
+                page_image, page_xywh, _ = image_from_page(
```

Comment:
That is another function which has to be moved to `core`, right?

Reply:
Exactly. But a function from the workspace module was used prior to this PR. Treat it as a reminder. No worries.

Reply:
Indeed, we would want adopters to _forget_ all about `workspace.resolve_image_as_pil` and use the 2 new functions instead. Maybe we can even _hide_ the less abstract functions completely?

Reply:
`workspace.resolve_image_as_pil` could be marked as deprecated and phased out in the coming releases.

> Maybe we can even hide the less abstract functions completely?

What do you mean with "hide"?

Reply:
Sounds good. Hide: that would have been renaming with leading underscore, but of course, we do not want existing implementations to break.